content/minfraud/sandbox-environment.mdx

```diff
@@ -0,0 +1,79 @@
+---
+draft: false
+title: Testing minFraud in the MaxMind Sandbox
+---
+
+import SandboxIntro from '/content/_sandbox-intro.mdx';
+import SandboxLicenseKey from '/content/_sandbox-license-key.mdx';
+import SandboxLimits from '/content/_sandbox-limits.mdx';
+import SandboxLearnMore from '/content/_sandbox-learn-more.mdx';
+
+<SandboxIntro />
+
+## Generate a License Key for the Sandbox
+
+<SandboxLicenseKey />
+
+## Build your integration
+
+Consult our [quick start guide](/minfraud/evaluate-a-transaction?lang=en) or our
+[list of client
+APIs](/minfraud/evaluate-a-transaction?lang=en#links-to-maxmind-client-apis) to
+develop your integration.
+
+## Point your integration to the Sandbox service endpoints
+
+Use the service endpoints for the Sandbox version of the web services, as
+listed in the table below:
+
+| Sandbox Service | HTTP Method | Endpoint                                             |
+| --------------- | ----------- | ---------------------------------------------------- |
+| Score           | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/score`    |
+| Insights        | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/insights` |
+| Factors         | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/factors`  |
+
+Remember to use the license key generated by your Sandbox account to
+authenticate your requests.
+
+## Test data for the Sandbox minFraud service
+
+<Alert type="info">
+The Sandbox will return risk scores and risk data, but this data is not
+accurate and should not be used to evaluate the efficacy of the minFraud
+scoring engine, or for geolocation or IP intelligence. It is designed
+for testing purposes only.
+</Alert>
+
+You may only submit approved test data to the Sandbox version of the minFraud
+services. At this time, the only approved test transaction data are the three IP
+addresses listed below. You can submit transactions to Sandbox minFraud services
+with any of the following IP addresses as the [`/device/ip_address`](/minfraud/api-documentation/requests?lang=en#schema--request--device__ip_address)
+input. You should receive a minFraud response with risk scores and risk data.
+The IP risk score output ([`/ip_address/risk`](/minfraud/api-documentation/responses?lang=en#schema--response--ip-address__risk))
+will contain the expected response below.
+
+| Test IP Address Input | Test IP Risk Score Output |
```

Comment:
@dlieou are these the correct expected responses? i was looking at your discussion of the test data and i think i must have pulled the wrong numbers.

Reply:
@christophermluna this is where we still have a discrepancy. Those are the numbers I am getting. The good news is whether I entered these IPs in the query form or via curl command, the results I am getting for each IP are the same. With that said, each day I tried the results seemed to be different (went down a bit). :(

Reply:
what i am asking is, i thought you guys were talking about a "medium" score, but in this test data there's just a high IP risk score and two low IP risk scores. is that correct? is it correct that two of the IPs should return a near-0 IP risk score, and one of them should return a value near 75?